packages/marketplace/src/components/pages/developer-edit-app/developer-edit-app.tsx

```diff
@@ -178,6 +178,29 @@ export const generateInitialValues = (appDetail: AppDetailModel | null, develope
   return initialValues
 }
 
+export const sanitizeAppData = (appData: CreateAppRevisionModel): CreateAppRevisionModel => {
+  const sanitizedAppData = appData
+  if (sanitizedAppData.description === '') {
```

Comment:
Small point but you can just go `if(!sanitizedAppData.description) ....` since empty strings are falsy

Reply:
fixed